cdap-master/src/test/java/co/cask/cdap/master/startup/ConfigurationCheckTest.java

```diff
@@ -62,4 +62,9 @@ public void invalidAuditTopicTest() {
   public void invalidLogPartition() {
     runConfigurationCheck(Constants.Logging.LOG_PUBLISH_PARTITION_KEY, "application1");
   }
+
+  @Test (expected = RuntimeException.class)
+  public void invalidHBaseDDLExecutorConfTest() {
```

Comment:
We have `ExtensionLoaderTest` https://github.com/caskdata/cdap/blob/release/4.2/cdap-app-fabric/src/test/java/co/cask/cdap/extension/ExtensionLoaderTest.java

However it  loads extensions from system classloader in unit tests.